platform/security/common/src/main/java/org/codice/ddf/security/common/Security.java

```diff
@@ -82,6 +87,17 @@
 
   private static final javax.security.auth.Subject JAVA_ADMIN_SUBJECT = getAdminJavaSubject();
 
+  @VisibleForTesting static final int MAX_RETRIES = 10;
```

Comment:
@lessarderic I have seen `@VisibleForTesting` before. If it is going to be part of our unit test practice, let's (1) disseminate that to all the developers and (2) maybe update our testing guidelines.